cfme/containers/provider/openshift.py

```diff
@@ -55,6 +55,26 @@ def view_value_mapping(self):
         return out
 
 
+class AlertsEndpoint(DefaultEndpoint):
+    """Represents Alerts Endpoint"""
+    name = 'alerts'
+
+    @property
+    def view_value_mapping(self):
+        if current_version() > '5.9':
+            out = {
+                'hostname': self.hostname,
+                'api_port': self.api_port
+            }
+            if current_version() >= '5.8':
```

Comment:
__REQUIRED__
I'm not sure what is the logic here but something is wrong - if you visit this line `current_version() >= '5.8'` is True for sure since `current_version() > '5.9'`. It looks like a mistake - please fix this

Reply:
Fixed, good catch